src/soilbiogeochem/SoilBiogeochemPotentialMod.F90

```diff
@@ -320,20 +320,6 @@ subroutine SoilBiogeochemPotential (bounds, num_soilc, filter_soilc, &
             potential_immob_vr(c,j) = immob(c,j)
          end do
       end do
-   else  ! use_fates
-      ! As a first step we are making this a C-only model, so no N downregulation of fluxes. 
-      do k = 1, ndecomp_cascade_transitions
-         do j = 1,nlevdecomp
-            do fc = 1,num_soilc
-               c = filter_soilc(fc)
-               !
-               p_decomp_cpool_loss(c,j,k) = decomp_cpools_vr(c,j,cascade_donor_pool(k)) &
-                    * decomp_k(c,j,cascade_donor_pool(k))  * pathfrac_decomp_cascade(c,j,k)
-               !
-            end do
-         end do
-      end do
-   end if
 
```

Comment:
thy is this getting removed for FATES cases?  I'm assuming it's because you're providing supplemental N instead?